src/Pages/Detail/DetailNav.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import "./DetailNav.scss";
+
+const NAV_ID = [
+  { id: "공간소개" },
+  { id: "시설안내" },
+  { id: "유의사항" },
+  { id: "환불정책" },
+  { id: "이용후기" },
+];
+
+class DetailNav extends Component {
+  componentDidMount() {
+    window.addEventListener("scroll", this.handleScroll);
+  }
+
+  handleScroll = () => {};
+
+  render() {
+    return (
+      <div className="DetailNav">
+        <ul className="navWrapper">
+          {NAV_ID.map((el) => {
+            return (
+              <li
+                onScroll={this.handleScroll}
+                className="navButton"
+                key={el.id}
+              >
+                <a href="#">{el.id}</a>
+              </li>
+            );
+          })}
```

Comment:
위에 NAV_ID 너무 잘 쓰셨어요! 이것도 참고 해주세요!
이 데이터는 절대 바뀔리가 없죠? 그냥 카테고리 같은 텍스트 모음 이어서 바뀔리가 없어요!
map 함수의 두 번째 인자가 index인데 이걸 이용해서 key 값을 줄 때 index로 줘도 돼요! 
그럼 전부 index로 주면 되는데 왜 굳이 id로 줘야 하냐!
만약에 배열의 데이터가 중간 부분이 바뀌면 그 index 값들이 전부 바뀌니까 key 값도 바뀌게 돼요
그럼 리액트에서 요소 하나를 특정할 때 한 번 더 로직이 실행되서 비효율적입니다!
그래서 key에는 그 배열이 바뀐다면 index를 key로 사용하시면 안 좋습니다!
다음 부터는 이거 생각해서 해주세요!
```suggestion
          {NAV_ID.map((el, index) => {
            return (
              <li
                onScroll={this.handleScroll}
                className="navButton"
                key={index}
              >
                <a href="#">{el}</a>
              </li>
            );
          })}
```

Reply:
네!!